build.sbt

```diff
@@ -1134,6 +1134,7 @@ lazy val runtime = (project in file("engine/runtime"))
     // Note [Unmanaged Classpath]
     Compile / unmanagedClasspath += (`core-definition` / Compile / packageBin).value,
     Test / unmanagedClasspath += (`core-definition` / Compile / packageBin).value,
+    Test / unmanagedClasspath += file("../../app/gui/view/graph-editor/src/builtin/visualization/native/inc"),
```

Comment:
I'm not sure what this is relative to. Can you please redefine it with the `baseDirectory` function (or something similar) instead of using the relative path https://www.scala-sbt.org/1.x/docs/Howto-Customizing-Paths.html